app/src/main/java/com/example/sharingang/items/ItemsAdapter.kt

```diff
@@ -53,11 +50,11 @@ class ItemsDiffCallback : DiffUtil.ItemCallback<Item>() {
 
 // Replace the description with its ID once the database is set
 class ItemListener(
-    val onEditListener: (item: Item) -> Unit,
+    //val onEditListener: (item: Item) -> Unit,
```

Comment:
We should remove all of these comments, no?

Reply:
My bad, I forgot to remove it apparently.